src/_P037_MQTTImport.ino

```diff
@@ -63,6 +63,8 @@ void Plugin_037_update_connect_status() {
   }
   if (MQTTclient_037_connected != connected) {
     MQTTclient_037_connected = !MQTTclient_037_connected;
+    P037_MQTTImport_connected  = MQTTclient_037_connected;
```

Comment:
This value must be reset to false when the plugin is deleted. 
It should be implemented in `case PLUGIN_EXIT:` , but it looks like that isn't even available it seems.

Reply:
I am not sure... 
The new variable (P037_MQTTImport_connected ) is set when MQTT disconnects, so the disconnection of MQTT is already happening somewhere.
Is not set when the plugin is initialized or destroyed.
I am just copying the value to a global variable (P037_MQTTImport_connected).


```
void Plugin_037_update_connect_status() {
  bool connected = false;
  if (MQTTclient_037 != NULL) {
    connected = MQTTclient_037->connected();
  }
  if (MQTTclient_037_connected != connected) {
    MQTTclient_037_connected = !MQTTclient_037_connected;
    P037_MQTTImport_connected  = MQTTclient_037_connected;  **<========**

    if (Settings.UseRules) {
      String event = connected ? F("MQTTimport#Connected") : F("MQTTimport#Disconnected");
      rulesProcessing(event);
    }
```